src/index.html

```diff
@@ -13,6 +13,53 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <!-- <h1>Stars</h1> -->
+    <div class="stars stars--0">
```

Comment:
It seems like you're using `div` tags for everything. While `div` is a very versatile tag, it doesn't provide any semantic information about its content. Consider using more semantic tags like `section` or `main` for grouping larger blocks of content, and `article` for individual, self-contained pieces of content. This will make your code more readable and accessible.